### ISSUE TITLE
[Accessibility][Progress Bar] Progress bars lack accessible names.

### ISSUE THREAD

**Issue by LevelAccessA11y** [adobe/aem-core-wcm-components#1728]
**Steps to Reproduce**
1. Open page in Google Chrome.
2. Enable NVDA screen reader.
3. Read progress bar element.

**Current Behavior**
The progress bar values are announced by a screen reader, but no context or accessible name is provided.

**Expected behavior/code**
Progress bars have meaningful accessible names.

**Environment**
Chrome 91, Windows 10, NVDA 2020.4 
AEM Core Components v. 2.17  

**Affected Users**
Screen reader users

**WCAG SC**
4.1.2 Name, Role, Value

**Possible Solution**
Ensure that progress bars have meaningful accessible names by using an aria-labelledby attribute if appropriate label text is visible or aria-label attribute if not visible text is available.

**Comment by richardhand** [comment]
@adobe export issue to Jira project SITES
